Extensions/BitmapText/JsExtension.js

```diff
@@ -0,0 +1,646 @@
+      const opacity = properties.get('opacity').getValue();
+      if (opacity !== this._pixiObject.opacity) {
+        this._pixiObject.alpha = opacity / 255;
+      }
+
+      //NOTE color - bitmapFont
+      const color = properties.get('color').getValue();
+      if (color !== this.style.fill) {
+        this.style.fill = color;
+        this._pixiObject.dirty = true;
+      }
+
+      const fontSize = Number(properties.get('fontSize').getValue());
+      if (fontSize !== this.style.fontSize) {
+        this.style.fontSize = fontSize; //size of the bitmapFont, generate a texture
+        this._pixiObject.fontSize = fontSize; // size of the bitmapText
+        this._pixiObject.dirty = true;
+      }
+
+      //NOTE fontFamily - bitmapFont
+      const fontResourceName = properties.get('fontFamily').getValue();
+      if (this._fontResourceName !== fontResourceName) {
+        this._fontResourceName = fontResourceName;
+
+        this._pixiResourcesLoader
+          .loadFontFamily(this._project, fontResourceName)
+          .then(fontFamily => {
+            // Once the font is loaded, we can use the given fontFamily.
+            this.style.fontFamily = fontFamily;
+            this._pixiObject.dirty = true;
+          })
+          .catch(err => {
+            // Ignore errors
+            console.warn(
+              'Unable to load font family for RenderedBitmapTextInstance',
+              err
+            );
+          });
+      }
+
+      const width = this._instance.hasCustomSize()
+        ? this._instance.getCustomWidth()
+        : this.getDefaultWidth();
+      const height = this._instance.hasCustomSize()
+        ? this._instance.getCustomHeight()
+        : this.getDefaultHeight();
+
+      //NOTE wordWrap to maxWidth - bitmapText
+      const wordWrap = properties.get('wordWrap').getValue() === 'true';
+      if (wordWrap && this._instance.hasCustomSize()) {
+        this._pixiObject.maxWidth = width;
+        this._pixiObject.dirty = true;
+      } else {
+        this._pixiObject.maxWidth = 0;
+        this._pixiObject.dirty = true;
+      }
+
+      // NOTE align - bitmapText
+      const align = properties.get('align').getValue();
+      if (align !== this._pixiObject.align) {
+        this._pixiObject.align = align;
+      }
+
+      this._pixiObject.position.x = this._instance.getX() + width / 2;
+      this._pixiObject.position.y = this._instance.getY() + height / 2;
+      this._pixiObject.rotation = RenderedInstance.toRad(
+        this._instance.getAngle()
+      );
+
+      const slugFontName =
+        this.style.fontFamily +
+        '-' +
+        this.style.fontSize +
+        '-' +
+        this.style.fill +
+        '-bitmapFont';
+
+      if (!PIXI.BitmapFont.available[slugFontName] || this._pixiObject.dirty) {
+        PIXI.BitmapFont.from(slugFontName, this.style, {
+          chars: PIXI.BitmapFont.ASCII,
+          textureWidth: 1024,
+        });
+        this._pixiObject.dirty = true;
+      }
```

Comment:
This seems to be called repeatedly without interruption - add a console.log inside the if and check if it's called multiple times.
You can tell that because there is very high CPU usage and if you launch the Chrome profiler, you can see huge calls to BitmapFont.from (which is a terribly expensive function, as it's generating a font in memory!):
![image](https://user-images.githubusercontent.com/1280130/91648768-6a0b2a80-ea63-11ea-81c3-3840580c7976.png)